sdk/core/core-amqp/CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # Release History
 
-## 2.0.1 (Unreleased)
+## 2.1.0 (Unreleased)
+
+- Adds the ability to configure the `hostname` and `port` that a `ConnectionContextBase` will use
```

Comment:
Can we add something to clarify the difference between the `host` and `hostname` configurations?

Reply:
Updated!